app/views/admin/api_docs/base/swagger.html.erb

```diff
@@ -63,7 +63,8 @@
         dom_id: "swagger-ui-container",
         supportedSubmitMethods: ['get', 'post', 'put', 'delete', 'patch'],
         onComplete: function(swaggerApi, swaggerUi) {
-          $('#swagger-ui-container pre code').each(function(i, e) {
+          document.querySelectorAll('#swagger-ui-container pre code').forEach((e) => {
```

Comment:
IMO, we can remove this `onComplete` block.

I've been looking at this, and I think it only affects highlighting some examples/request schemas in Swagger UI v2.
Example:

with this block:
![Screenshot from 2025-02-27 13-11-59](https://github.com/user-attachments/assets/76a86352-0232-40fa-b46f-ddc122f1ba26)

without this block:

![Screenshot from 2025-02-27 13-13-30](https://github.com/user-attachments/assets/a0e76480-1847-4825-876a-6029afabc770)

I don't think it's a big hit... The response code is highligted in any case, because I think it's part of `swagger-ui.js` mentioned in the comment below.

Reply:
oh, I didn't notice it's merged already :laughing: